get_sbu_data.py

```diff
@@ -0,0 +1,167 @@
+from ECHO_modules.geographies import spatial_tables, fips, region_field, states # Import for mapping purposes
+import ssl
+ssl._create_default_https_context = ssl._create_unverified_context
+
+# Styles for States ("other") and selected regions (e.g. Zip Codes) - "this"
+map_style = {'this': {'fillColor': '#0099ff', 'color': '#182799', "weight": 1},
+'other': {'fillColor': '#FFA500', 'color': '#182799', "weight": 1}}
+
+selected_state = st.selectbox(
+    'Select a state',
+    states)
+
+# Get list of counties for selected state
+url = "https://raw.githubusercontent.com/edgi-govdata-archiving/"
+url += "ECHO_modules/packaging/data/state_counties_corrected.csv"
+all_counties = pd.read_csv( url )
+counties = all_counties[all_counties['FAC_STATE'] == selected_state]['County']
+counties = counties.unique()
+
+selected_county = st.selectbox(
+    'Select a county',
+    counties)
+
+st.write('You selected:', selected_county, ", ", selected_state)
+
+def format_county(selected_county):
+    '''
+    Returns the formatted county string from uppercase to lowercase and capitalized
+    Example: format_county('SAN FRANCISCO') returns 'San Francisco'
+    '''
+    res = ''
+    county_words = selected_county.split(' ')
+    for word in county_words:
+        res += word.lower().capitalize() + ' '
+    return res[:-1]
+
+county_name = format_county(selected_county)
+
+
+st.header('Get Facilities in Selected County')
+#sql = 'select * from "ECHO_EXPORTER" where "FAC_COUNTY" = \'TOMPKINS\' and "FAC_STATE" = \'NY\'' # "ECHO_EXPORTER" contains basic info about all regulated industrial facilities, including location
+#fac = get_echo_data(sql)
+"""Note: because of errors in the EPA's own data, the above query may not always return accurate results.
+For instance, sometimes county names are listed as just "Tompkins"; sometimes they are listed as "Tompkins County".
+A more robust version utilizes another approach. See below:
+"""
+# Get county names based on Steve's work. If we know our county of interest is Tompkins, NY, we can search for other names it may be listed under
+counties = pd.read_csv("https://raw.githubusercontent.com/edgi-govdata-archiving/ECHO_modules/main/data/state_counties_corrected.csv") # Get a county name lookup
+counties = counties.groupby(by=["FAC_STATE", "County", "FAC_COUNTY"]).count() # Restructure the data
+counties = counties.iloc[(counties.index.get_level_values('County') == selected_county) & (counties.index.get_level_values('FAC_STATE') == selected_state)].reset_index() # Search for COUNTY, STATE
+counties = list(counties["FAC_COUNTY"].unique()) # List FAC_COUNTY names in ECHO associated with COUNTY
+# Format a sql query
+c = "("
+for county in counties:
+  c += "\'"+county+"\',"
+c = c[:-1] + ")"
+sql = 'select * from "ECHO_EXPORTER" where "FAC_COUNTY" in ' + c + ' and "FAC_STATE" = \'' + selected_state + '\'' # "ECHO_EXPORTER" contains basic info about all regulated industrial facilities, including location
+print(sql)
```

Comment:
Please correct me if I am wrong, but do you need this print statement? Was this for debugging?